db/migrate/20170601064430_change_column_of_receivers.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnOfReceivers < ActiveRecord::Migration[5.0]
+  def change
+    change_column :receivers, :status, 'integer USING CAST(status AS integer)'
```

Comment:
same here.
It should be just:
`change_column :receivers, :status, :integer, default: 0`